### ISSUE TITLE
Issue with String Replace Method

### ISSUE THREAD

**Issue by objeck** [objeck/objeck-lang#68]
The following code crashes

```
class Test {
	function : Main(args : String[]) ~ Nil {
		a := "FooBar";
		a := a->Replace("Foo", "");
		a->PrintLine();
	}
}
```

**Comment by objeck** [comment]
Fixed in 69c52cc70ae1b05202fb56df5967de9b005adf36

**Comment by objeck** [comment]
Released in e41e1be6bbf622722fab78fc0a0ef014aec3fd0a
